changelog/dev-phpstan-for-woopayments

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+PHPStan code analyser for WooPayments.
```

Comment:
Technically, this is not a part of what gets released to .org. Do we need a changelog entry, or just a comment?

Reply:
Just a comment will suffice. Let me change this.